scipy/stats/_multivariate.py

```diff
@@ -1720,6 +1689,14 @@ class wishart_gen(multi_rv_generic):
     The input quantiles can be any shape of array, as long as the last
     axis labels the components.
 
+    Alternatively, the object may be called (as a function) to fix the degrees
+    of freedom and scale parameters, returning a "frozen" Wishart random
+    variable:
+
+    >>> rv = wishart(df=1, scale=1)
+    >>> # Frozen object with the same methods but holding the given
+    >>> # degrees of freedom and scale fixed.
+
```

Comment:
![image](https://user-images.githubusercontent.com/6570539/154623956-2990ccb7-613d-4256-b1ba-48939e869e4d.png)

The plot is showing at the end. Do we need `plt.show()` above or something?

Reply:
> Do we need `plt.show()` above or something?

I will try it out locally.

Reply:
Done. This fixes it.

Reply:
Needed in `multivariate_normal` and `invwishart`,

Reply:
Done.